client/payment-details/dispute-details/dispute-awaiting-response-details.tsx

```diff
@@ -146,7 +150,9 @@ function getAcceptDisputeProps( dispute: Dispute ): AcceptDisputeProps {
 				),
 			},
 		],
-		modalButtonLabel: __( 'Accept dispute', 'woocommerce-payments' ),
+		modalButtonLabel: isLoading
```

Comment:
`isLoading` could provide more info about what is loading. I think renaming is worthwhile because `isLoading` sounds like a pre-display state, not a "submitting" state. I realise this is an existing name in parent component, I think it's useful to rename there to be clear about what is loading.

The longest, least ambiguous name I can think of is `isAcceptDisputeSubmitPending` – that might be too much 😁

Reply:
Renamed to `isDisputeAcceptRequestPending` in b60c8dc.